mathesar/views/frontend.py

```diff
@@ -16,7 +16,7 @@ def get_common_data(request):
     }
 
 
-def index(request):
+def index(request, **kwargs):
```

Comment:
We should redirect the `index` to the first database available so that it doesn't load a blank page at the root.

Reply:
Currently the redirection is done on the client. It doesn't incur additional calls to the server, since the preloaded database info is present. Do we need to do this on the server?

Reply:
when I ran it locally, it did not redirect, I had to go to `localhost:8000/mathesar/` in the URL bar to see any data.

it should be pretty simple to do this on the server, you can use a Django `redirect` in the index view. https://docs.djangoproject.com/en/3.2/topics/http/shortcuts/#redirect. I'd like to keep things on the server if they don't _need_ to be on the client.

Reply:
Okay, I need to look into why it did not redirect. I will however make the changes on the server as you mentioned.

Reply:
I've made this change. Please take a look @kgodey